.github/workflows/containerize-spring-boot-jibber.yml

```diff
@@ -1,21 +1,21 @@
-name: native-image/containerize
+name: native-image/containerize/spring-boot-microservice-jibber
 on:
   push:
     paths:
-      - 'native-image/containerize/**'
-      - '.github/workflows/native-image-containerize.yml'
+      - 'native-image/containerize/spring-boot-microservice-jibber/**'
```

Comment:
Yesterday we published the new 24EA build and now the demo check fails on it with:

```
Run cd native-image/containerize/spring-boot-microservice-jibber
WARNING: A restricted method in java.lang.System has been called
WARNING: java.lang.System::load has been called by org.fusesource.jansi.internal.JansiLoader in an unnamed module (file:/usr/share/apache-maven-3.8.8/lib/jansi-2.4.0.jar)
WARNING: Use --enable-native-access=ALL-UNNAMED to avoid a warning for callers in this module
WARNING: Restricted methods will be blocked in a future release unless native access is enabled

WARNING: A terminally deprecated method in sun.misc.Unsafe has been called
WARNING: sun.misc.Unsafe::objectFieldOffset has been called by com.google.common.util.concurrent.AbstractFuture$UnsafeAtomicHelper (file:/usr/share/apache-maven-3.8.8/lib/guava-25.1-android.jar)
WARNING: Please consider reporting this to the maintainers of class com.google.common.util.concurrent.AbstractFuture$UnsafeAtomicHelper
WARNING: sun.misc.Unsafe::objectFieldOffset will be removed in a future release
...
```
Related to the updated configuration for `Unsafe...`. That's my guess so far.